File: e-dashboard-ui/src/ClientApp/src/views/Diagram/DiagramLib/Shapes/Style.ts
import { IStyle } from "./interfaces";

export class Style implements IStyle {
  public opacity: number;
  public strokeWidth: number;
  public strokeStyle: string;
  public fillStyle: string;

  constructor() {
    this.opacity = 1;
    this.strokeWidth = 1;
    this.strokeStyle = "#000000";
    this.fillStyle = "#00000000";
  }
}

File: e-dashboard-api/src/Domain/Diagram/Shapes/Style.cs

namespace EnergyDashboard.Domain.Diagram.Shapes;

public class Style
{
    public Style()
    {
        this.Opacity = 1.0;
        this.FillStyle = "#00000000";
        this.StrokeStyle = "#000000";
        this.StrokeWidth = 1;
    }
    public double Opacity { get; set; }
    public int StrokeWidth { get; set; }
    public string StrokeStyle { get; set; }
    public string FillStyle { get; set; }
}
